tests/console/systemd_wo_udev.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright 2012-2018 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: systemd has no direct dependency for udev package
+# Maintainer: QA-C <qa-c@suse.de>
+# Tags: SLE-21856
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use utils qw(zypper_call);
+
+sub run {
+    shift->select_serial_terminal;
+    my $out = '/root/zypper_udev.out';
+
+    assert_script_run("rpm -q --requires systemd | tee $out");
+    zypper_call("install --force --dry-run --no-confirm --download-only systemd | tee -a $out", exitcode => [0, 4], dumb_term => 0);
+    script_run("grep -iv udev $out") == 0 or die 'systemd on sle15sp4+, leap15.4+ and TW should have no dependency to udev package!';
+}
```

Comment:
I would add for helping troubleshooting.
```
upload_logs($out, failok => 1);
```

Reply:
that can go to the post fail hook with zypper history, wdyt ?